app/src/main/java/com/example/habitsmasher/PublicPrivateButtons.java

```diff
@@ -41,20 +45,34 @@ public PublicPrivateButtons(View view){
      * @param isPublic Whether the habit is public or not.
      */
     public PublicPrivateButtons(View view, boolean isPublic){
-        // get the buttons
-        _publicButton = view.findViewById(R.id.public_button);
-        _privateButton = view.findViewById(R.id.private_button);
-
-        Log.d(TAG, String.valueOf(isPublic));
+        // get the buttons if they exist
+        try {
+            _publicButton = view.findViewById(R.id.public_button);
+            _privateButton = view.findViewById(R.id.private_button);
+        }
+        catch (Exception e){
+            Log.e(TAG, "PublicPrivateButtons: Buttons Not Found!", e);
+        }
```

Comment:
These are for the testing, was having errors without these lines. Basically just ignores the buttons when running the habit validator unit tests.